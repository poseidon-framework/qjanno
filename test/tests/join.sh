qhs -H -O -T "SELECT basic.foo,big.baz,big.bar FROM basic.csv basic JOIN big.csv big ON basic.baz = big.baz WHERE basic.bar IS NOT NULL"
