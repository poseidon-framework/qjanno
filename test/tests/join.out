foo	baz	bar
a0	a2	1
b0	b2	1
